web/src/scheduled_messages.js

```diff
@@ -74,6 +80,7 @@ export function remove_scheduled_message(scheduled_message_id) {
     );
     if (msg_index !== undefined) {
         scheduled_messages_data.splice(msg_index, 1);
```

Comment:
It's a bit weird that `scheduled_messages_data` is a list rather than a dictionary mapping IDs to objects; should we rework that?

Reply:
Opened #25557 and added to my TODO, I will do it unless someone beats me to it.